File: backend/src/main/java/com/marcio/springjwt/Entity/Educacion.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.marcio.springjwt.Entity;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import lombok.Getter;
import lombok.Setter;


@Entity
@Setter
@Getter

public class Educacion {
    
    @Id
    @GeneratedValue(strategy=GenerationType.IDENTITY)
    private Long id;
    private String urlImg;
    private String titulo;
    private String descripcion;

   
}

File: FrontEnd/src/app/model/educacion.ts
export class Educacion{
    id?:number;
    titulo: string;
    descripcion: string;
    urlImg: string;
    
    constructor( titulo:string, descripcion:string,urlImg:string){
       
         this.descripcion=descripcion;
         this.titulo=titulo;
         this.urlImg=urlImg;
        
    }
 
 }